docs/providers/aws/guide/variables.md

```diff
@@ -103,6 +103,28 @@ functions:
 
 In the above example, you're referencing the entire `myCustomFile.yml` file in the `custom` property. You need to pass the path relative to your service directory. You can also request specific properties in that file as shown in the `schedule` property. It's completely recursive and you can go as deep as you want.
 
+## Reference Variables in Javascript Files
+To add dynamic data into your variables, reference javascript files by putting `${file(./myFile.js):someModule}` syntax in your `serverless.yml`. This functionality is recursive, so you can go as deep in that file as you want. Here's an example:
```

Comment:
Could you add or extend the current example to show that you can also reference sub properties, so you can return a hash in the js file and then reference that sub property (same way you did in the test). Other than that great change!